server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -984,23 +1031,23 @@ private long balanceTablets() {
             continue;
           }
           migrationsOutForLevel++;
-          migrations.put(ke, TabletServerIdImpl.toThrift(m.getNewTabletServer()));
+          getContext().getAmple().mutateTablet(ke)
```

Comment:
This will do a single write RPC and wait for it. Would be better to to create a tablet mutor for once and use it for the entire loop.  This will allow much more efficient steaming of data to the write ahead logs on the tablet servers, instead of making a single RPC that does a single write ahead log write on the tablet server and waiting for it.